components/brave_federated_learning/brave_operational_profiling_features.h

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_BRAVE_OPERATIONAL_PROFILING_FEATURES_H_
+#define BRAVE_COMPONENTS_BRAVE_FEDERATED_LEARNING_BRAVE_OPERATIONAL_PROFILING_FEATURES_H_
+
+#include "base/feature_list.h"
+
+namespace operational_profiling {
+namespace features {
+
+extern const base::Feature kUserOperationalProfiling;
+
+bool IsOperationalProfilingEnabled();
+
+int GetCollectionSlotSizeValue();
+int GetSimulateLocalTrainingStepDurationValue();
+int GetEphemeralIdLifetime();
```

Comment:
IMO this name conflicts with ephemeral storage machinery, so maybe pick another one?

Reply:
How about `CollectionId`?

Reply:
`TemporaryCollectionId`

Reply:
yeah, either one is fine with me

Reply:
@moritzhaller highlighted how “temporary” is implied by the business logic and parameters like “lifetime”, so going for `CollectionId` and `collection_id`

Reply:
👍